docs/Backup & restore.md

```diff
@@ -87,22 +75,34 @@ Devices MUST allow restoration of modified backups (full or partial backups wher
 
 // TBD: Do we think some devices may need to reboot in order to apply a restore?
 
-## 3. Restoring the same device instance on a different device revision
-
-Devices can undergo device revision upgrades or downgrades during their deployment lifetime. This can lead to backups performed on a `device revision` having to be restored on a different one.
+## 3. Restoring on a device with an incompatible revision
 
-Restoring follows a similar workflow to [restoring the same device revision](#2-restoring-same-device-instance-and-the-same-device-revision).
+Restoring follows a similar [workflow](#2-restoring-on-a-device-with-a-compatible-revision) with the difference that the restore device does not have a compatible revision.
 
 | ![Restoring on a different device revision](images/restoring-same-dev-different-dev-rev.png) |
 |:--:|
 | _**Restoring on a different device revision**_ |
 
-Devices are RECOMMENDED to allow the restoration of backups created on a different device revision than their own.
-
 Devices MUST allow the partial restoration of backups which have at least one role path `status` of `Ok` when supplying the `allowPartial` argument of `true` in the request.
 
-## 4. Restoring a different instance of the same device type using the same device revision
+// TBD: Facilities can have a multitude of instances of the same device type. Backups performed on a device instance can be used to bootstrap other instances of the same device type. What if the new device overwrites things like IP addresses? Are we saying the restore should be used as a bootstrap mechanism (do we need a isTemplate boolean flag when restoring)?
 
-Facilities can have a multitude of instances of the same device type. Backups performed on a device instance can be used to bootstrap other instances of the same device type.
+## Definitions
+
+A `device`, for the purposes of this section, is a physical or logical entity that can be backed up and restored using the procedures described. It may or may not correspond to an IS-04 Device or IS-04 Node.
```

Comment:
Remove normative language usage.
```suggestion
A `device`, for the purposes of this section, is a physical or logical entity that can be backed up and restored using the procedures described. It is not necessarily the same entity as an IS-04 Device or IS-04 Node, but it can be.
```